src/protocol/packets/s2c.rs

```diff
@@ -629,6 +629,12 @@ pub mod play {
         }
     }
 
+    def_struct! {
+        SendActionBar {
```

Comment:
Should call this `OverlayMessage` for consistency with packets.json. I know it's dumb.

Reply:
It's not, though possibly for the def_struct macro for packets, I think using the names from JSON as a type hint / error would be good